### PR TITLE
[13.0][ADD] base_ir_actions_report_sequence

### PR DESCRIPTION
**Pull request by JaumeBforgeFlow**
This module adds the ability to choose the order in which reports are shown using a sequence.

### CODE REVIEW

**Review by LoisRForgeFlow — APPROVED** [review]
Code review :+1:

**Review by AaronHForgeFlow — APPROVED** [review]
LGTM

**Review by JasminSForgeFlow — APPROVED** [review]
LGTM!

**Comment by JordiMForgeFlow** [comment]
Code + functional review 👍🏼 

@JaumeBforgeFlow could you add the sequence as the first column in the tree view?

image

File: base_ir_actions_report_sequence/views/ir_actions_report_views.xml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    </record>
+    <record id="act_report_xml_view_tree_priority" model="ir.ui.view">
+        <field
+            name="name"
+        >ir.actions.report.tree - base_ir_actions_report_sequence</field>
+        <field name="model">ir.actions.report</field>
+        <field name="inherit_id" ref="base.act_report_xml_view_tree" />
+        <field name="arch" type="xml">
+            <field name="name" position="before">
+                <field name="priority" optional="hide" widget="handle" />
```
Comment (JordiMForgeFlow) [inline base_ir_actions_report_sequence/views/ir_actions_report_views.xml]:
```suggestion
                <field name="priority" optional="show" widget="handle" />
```

**Review by JordiMForgeFlow — APPROVED** [review]

**Pull request merged**